### DISCUSSION TITLE
機械系的未來

### DISCUSSION THREAD

**Discussion by Jerry10413** [1101-GS4719-C/Python-Community-GS4719-C#989] [General]
大家好像都聽說讀機械系未來不會沒有工作可以做，但是也不會很有錢，還有很多人覺得機械系就是黑手，我覺得根據這些，我來分享自己的看法，我自己是機械系大三，我目前看到系上的專題很多都是在做人機協作，像是智慧工廠，智慧產線的部分，我覺得這跟一般機械系差很多，機械系現在的走向是越來越往和AI結合的部份去了，而以現在的趨勢來看，AI也是一個未來會很夯的議題，所以我認為機械系並不會像是這些人內心的刻板印象，而是有再慢慢地轉型了~~

**Comment by wanderer657** [comment]
機械工程不僅與智慧製造和AI結合，還在能源、交通和醫療等領域發揮重要作用。舉例來說，在能源方面，機械工程師可以開發新型節能設備和可再生能源技術；在交通領域，他們設計更安全、更節能的交通工具；在醫療領域，則開發醫療設備和生物醫學技術。這些都展示了機械工程的多樣性和廣泛性，顯示出其在不同領域中的潛力和重要性。

**Comment by 20050201Shawn** [comment]
有時看低卡跟ptt的機械系出路文跟勸退文真的會心一沉，覺得未來只有待工廠跟車行的分。但我認為機械產業是有在慢慢轉型的，並且機械工程在各領域中都是有發展空間的，為一切工業之基礎。只能說有甚麼技術能力不是最重要的，最重要的還是解決問題的能力，甚至是改良問題。當升級到工業4.0，機械工業可以朝著更人性、更智慧的方向去發展，像是人本交通、電子支付、綠色科技等等，都可使機械產業達到永續且完整的經營。

這只是一個我(沒出社會的大一)小小的淺見。

**Comment by YifenLin** [comment]
其實機械系涵蓋的領域很廣，機構設計、材料、控制都涵蓋機械系的範疇。我之前聽一個學長分享過，早期沒有沒有材料系，是從機械分支出來的(這部分我是沒有求證過拉)。機械系這樣的好處是可以學得很廣，不過相較起來就沒辦法學的深(我認為這可能是網路上大家不看好機械系的原因之一)。其實只要選定自己喜歡的領域，並且專精於他，就會在那個領域一樣擁有競爭力!

**Comment by jeff110303549** [comment]
我覺得很多人一想到機械系的出路可能都會想到黑手(可能機械系自己也是...？) 應該只是基於對這個科系的不了解，畢竟機械系的出路本身就在傳統產業及科技產業之間，也是蠻容易讓人產生刻板印象的；所以中央的機械把學生細分成三個組別除了展現了這門科系所包含的多樣性外，更是讓學生們專精在一個大領域的方法。談到人工智慧，未來也必然是機械系學生能夠學習的一個方向，目前我知道中央機械已經開設了碩士班人工智慧應用組，期望在不遠的未來也能看到大學部也開設和人工智慧、智慧製造相關的班級吧！

**Comment by benbebobo149** [comment]
我本身是讀資訊工程系，我反而覺得現今讀資工出來的，競爭力就不會那麼高，第一個是讀資電的人爆炸性的成長，所以不太知道等我開始工作後，是否還有紅利可以嘗到。第二個是我們系上的課程，著重在理論還是較多，比較少結合應用的部分。但是機械系或是其他系所，除了自己本身的原系的專業外，跨領域學習AI等等，未來是還有很大的成長空間。

**Comment by wanka2030** [comment]
當我已為機械系的出魯只能在工廠鎖螺絲、修設備的時候，早就有很多學長或這在業界上更多的人才能夠在各方領域留下不同的痕跡。雖然說機械系的課程規劃又臭又長又繁雜，但每堂課都能夠學習到有關於生活上機械設備上有關的知識，大至隨處可見的螢幕、LED燈，小至妳同學早上不小心落鏈的腳踏車鍊條，皆是有關於機械系所學的。不怕沒有吃飯的地方，機械系應該還不是黃昏科系，就此來看的話。

**Comment by yyujimmy** [comment]
提到的觀點很有趣，也讓人重新思考機械工程的發展方向。確實，隨著技術的進步和產業的變革，機械工程領域正在朝著更加智能化和自動化的方向發展。人機協作、智慧工廠和智慧產線等概念正成為機械工程的熱門研究領域，這些領域的發展需要機械工程師與人工智能專家密切合作，共同推動技術的進步。

機械系的轉型確實可以讓學生們在未來的就業市場中更具競爭力。而且，隨著AI技術的發展，機械工程師可能會在更廣泛的領域中找到機會，例如機器學習、數據分析和自動化系統的設計。因此，機械系未來的前景可能比人們想像的更加廣闊和多樣化。

**Comment by KH1126** [comment]
機械系涵蓋的領域太廣，要修的課也很多。
我覺得專精自己喜歡的領域的話，提升競爭力，不怕找不到工作。

**Comment by YOGA012** [comment]
我也是機械系的學生，對你說的這些深有同感。很多人對機械系的刻板印象確實還停留在傳統的製造業和操作機械上，但其實我們學到的東西遠不止於此。現在很多專題和課程都在強調智慧工廠和智慧產線，尤其是與AI技術的結合，這讓我們的學習和未來的發展空間更廣闊。機械系正在逐步轉型，未來的機會其實非常多元，並不是大家想像中的那麼單一或局限。

**Comment by titan92601** [comment]
我認為你的觀點非常好，對機械工程的理解也非常透徹。確實，許多人對機械系存在一些刻板印象，認為機械系畢業生的工作多半是體力勞動，或者薪資不高。然而，隨著技術的發展，機械工程的範疇和應用領域也在不斷擴展，特別是在與AI和自動化技術相結合的方面。
